src/components/App.js

```diff
@@ -12,58 +12,61 @@ import { comparePages } from '../utility/compare'
 import { LanguageContext } from './Translate'
 
 import './App.css'
+import { addLangToUrl, useLang } from '../utility/lang'
 
-const createLinks = nav => nav
+const createLinks = (nav, lang) => nav
   .sort(comparePages)
-  .map(({ slug, title }) => <Link key={slug} to={slug}>{title}</Link>)
+  .map(({ slug, title }) =>
+    <Link key={slug} to={addLangToUrl(slug, lang)}>{title}</Link>
+  )
 
-const renderMethone = path =>
-  <Taitan pathname={path}>
-    {({ nav }) =>
+const renderMethone = (path, lang) => {
+  const info = {
+    "en": "Svenska",
+    "sv": "English"
+  }[lang || 'sv'] || "Svenska";
```

Comment:
There is also a `$DEFAULT_LANG` env variable for taitan. Perhaps it's bad to have a default at both places?

Reply:
yeah, not completely happy with how I implemented this, will take a look at it. I don't think we can get away from having some sort of default in both repos unless we do some more refactoring, but it can be cleaner than this for sure.

Reply:
We now have a much more implicit default here. But do note that this is just for the language switching logic, so that you can always get back to a valid language, so I think it is fine that we have a "double default" in this case.

An alternative way could be to add a endpoint to taitan for fetching the default lang, but I'm unsure if I liek that, and wither way I think it should wait for another PR in that case.